docker/runtime/doris-compose/cluster.py

```diff
@@ -300,6 +299,7 @@ def docker_env(self):
             "DORIS_HOME": os.path.join(self.docker_home_dir()),
             "STOP_GRACE": 1 if enable_coverage else 0,
             "IS_CLOUD": 1 if self.cluster.is_cloud else 0,
+            "SQL_MODE_NODE_MGR": 1 if hasattr(self.cluster, 'sql_mode_node_mgr') and self.cluster.sql_mode_node_mgr else 0
```

Comment:
Cluster init function contains sql_mode_node_mgr,  it always has attr sql_mode_node_mgr even for the old clusters.  Delete check "hasattr(self.cluster, 'sql_mode_node_mgr')"